TypeCobol/Compiler/CupPreprocessor/CompilerDirectiveBuilder.cs

```diff
@@ -91,6 +91,12 @@ public virtual void EnterCopyCompilerStatement(TypeCobol.Compiler.Scanner.Token
             bool isCopy = copy.COPYToken.TokenType == TokenType.COPY;
             copy.TextName = GetName(qualifiedTextName.TextName);
             copy.TextNameSymbol = qualifiedTextName.TextName;
+            if (suppress != null)
+            {
+                Diagnostic error = new Diagnostic(MessageCode.Warning, suppress.Column, suppress.EndColumn, 
+                    suppress.Line, "\"COPY SUPPRESS\" should not be used");
+                CompilerDirective.AddDiagnostic(error);
+            }
```

Comment:
Could you please move this code down to line 144 to mirror the code in the other CompilerDirectiveBuilder file ? This would allow easier comparison between the two.

Reply:
fixed; new commit
https://github.com/TypeCobolTeam/TypeCobol/tree/68d3be7fd420c735f36d49da44e7f495213c4bc0